src/main/java/org/scijava/ui/swing/script/TextEditor.java

```diff
@@ -2454,4 +2453,15 @@ public void changedUpdate(final DocumentEvent e) {
 		setTitle();
 	}
 
+	public void setFontSize(final float size) {
```

Comment:
Since the public API changed, I suggest that the version of this component be updated to `0.4.0-SNAPSHOT` in `pom.xml`.